src/fixes.rs

```diff
@@ -91,8 +91,8 @@ pub fn get_fixes_without_resolving_overlapping(
             if let Some(import_paths) = import_addition_paths {
                 let imports_suggestion = import_paths
                     .iter()
-                    .map(|import_path| format!("use {};", import_path))
-                    .join("\n");
+                    .map(|import_path| format!("use {};\n", import_path))
```

Comment:
why not `filter_map` here? the problem was empty strings being joined by `\n` - right?

Reply:
No, the problem was that with single import, it wasn't adding the `\n`